.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "packages/chatpal-search"]
+	path = packages/chatpal-search
+	url = https://github.com/assistify/chatpal-search.git
```

Comment:
Added the chatpal repository as subpackage. However, I have no clue what this really means and just hope all is good. All changes of the chatpal have been pushed to this repo